### PR TITLE
Add release notes for c4b-azure 0.18.1

### PR DESCRIPTION
**Pull request by Windos**
## Description Of Changes

Add release notes for c4b-azure 0.18.1

## Motivation and Context

Publishing release notes for new release.

## Testing

* [X] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

image

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [X] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

**Comment by Windos** [comment]
@corbob or @vexx32 if either of you get a chance to review this PR I'd muchly appreciate it!

File: input/en-us/c4b-environments/azure/release-notes.md [hunk 1]
```diff
@@ -13,6 +13,32 @@ TwitterImage: https://img.chocolatey.org/social-share/release-notes-chocolatey-f
 >
 > Issue links may not be publicly available at this time.
 
+## 0.18.1 (May 25, 2023)
+
+### Bug Fix
+
+- Updating Jenkins breaks AppGateway Health Checks - see [#126](https://github.com/chocolatey/c4b-azure/issues/126)
```
Comment (pauby) [inline input/en-us/c4b-environments/azure/release-notes.md:20]:
```suggestion
- Fix - Updating Jenkins breaks AppGateway Health Checks - see [#126](https://github.com/chocolatey/c4b-azure/issues/126).
```

Comment (pauby) [reply input/en-us/c4b-environments/azure/release-notes.md:20]: The release notes format requires we add a 'Fix - ' at the start.

File: input/en-us/c4b-environments/azure/release-notes.md [hunk 2]
```diff
@@ -13,6 +13,32 @@ TwitterImage: https://img.chocolatey.org/social-share/release-notes-chocolatey-f
...
+
+### Improvements
+
+- Update Chocolatey software components to the latest available versions - see [#128](https://github.com/chocolatey/c4b-azure/issues/128)
+- Update ARM Template Resource API Versions - see [#129](https://github.com/chocolatey/c4b-azure/issues/129)
+- Update Jenkins Plugins and match selection in C4B Quick Start - see [#131](https://github.com/chocolatey/c4b-azure/issues/131)
```
Comment (pauby) [inline input/en-us/c4b-environments/azure/release-notes.md:26]:
```suggestion
- Update Chocolatey software components to the latest available versions - see [#128](https://github.com/chocolatey/c4b-azure/issues/128).
- Update ARM Template Resource API Versions - see [#129](https://github.com/chocolatey/c4b-azure/issues/129).
- Update Jenkins Plugins and match selection in C4B Quick Start - see [#131](https://github.com/chocolatey/c4b-azure/issues/131).
```

**Review by pauby — CHANGES_REQUESTED** [review]

**Review by pauby — APPROVED** [review]

**Review by corbob — APPROVED** [review]

**Pull request merged**